tests/sklearn/test_sklearn_model_export.py

```diff
@@ -768,12 +777,18 @@ def test_log_model_with_code_paths(sklearn_knn_model):
         add_mock.assert_called()
 
 
-def test_log_predict_proba(sklearn_logreg_model):
-    model, inference_dataframe = sklearn_logreg_model
-    expected_scores = model.predict_proba(inference_dataframe)
+@pytest.mark.parametrize(
+    "predict_fn", ["predict", "predict_proba", "predict_log_proba", "predict_joint_log_proba"]
+)
+def test_log_model_with_custom_pyfunc_predict_fn(sklearn_gaussian_model, predict_fn):
+    if Version(sklearn.__version__) < Version("1.2.0") and predict_fn == "predict_joint_log_proba":
+        pytest.skip("predict_joint_log_proba is not available in scikit-learn < 1.2.0")
```

Comment:
/ai teach me a better way to write this code using `pytest.param`.

Reply:
hmm I think this is less clear...?

Reply:
Not a blocker, I'd use `pytest.param` in this case to simplify the test function body.

Reply:
Imo adding ^ block to remove simple 2 lines if-statement doesn't pay off....